src/en/guide/conf/config/logging.adoc

```diff
@@ -1,5 +1,10 @@
-Since Grails 3.0, logging is handled by the http://logback.qos.ch[Logback logging framework] and can be configured with the `grails-app/conf/logback.xml` file.
+Since Grails 3.0, logging is handled by the http://logback.qos.ch[Logback logging framework] and can be configured with the `grails-app/conf/logback-spring.xml` file.
 
-NOTE: Since Grails 5.1.2 support for groovy configuration (`grails-app/conf/logback.groovy`) has been removed (by logback 1.2.9). It is possible to add back groovy configuration by adding the https://github.com/virtualdogbert/logback-groovy-config[logback-groovy-config] library to your project.
+- See the https://docs.spring.io/spring-boot/reference/features/logging.html[Spring Boot Logging] for all the available configuration options.
 
-For more information on configuring logging refer to the http://logback.qos.ch/manual/groovy.html[Logback documentation] on the subject.
\ No newline at end of file
+// To be re-included when a Grails 7 release of the plugin gets released
+// See: virtualdogbert/logback-groovy-config#15
+//
+// NOTE: Since Grails 5.1.2 support for groovy configuration (`grails-app/conf/logback.groovy`) has been removed (by logback 1.2.9). It is possible to add back groovy configuration by adding the https://github.com/virtualdogbert/logback-groovy-config[logback-groovy-config] library to your project.
+
```

Comment:
I think it would be beneficial to include an example logback-spring.xml file in this section and explain     `<springProfile name="development">`, since that is the main advantage.

```
<?xml version="1.0" encoding="UTF-8"?>
<configuration>
    <include resource="org/springframework/boot/logging/logback/defaults.xml"/>
    <include resource="org/springframework/boot/logging/logback/console-appender.xml"/>
    <root level="ERROR">
        <appender-ref ref="CONSOLE"/>
    </root>

    <springProfile name="development">
        <logger name="StackTrace" level="ERROR" additivity="false"/>
        <logger name="your.package" level="DEBUG"/>
        <logger name="org.hibernate.orm.deprecation" level="OFF"/>
        <root level="WARN"/>
    </springProfile>
</configuration>
```